tracer/test/snapshots/Security.AspNetCore2.SecurityEnabled.__test=blocking_url=_.verified.txt

```diff
@@ -25,7 +25,6 @@
       network.client.ip: 127.0.0.1,
       runtime-id: Guid_1,
       span.kind: server,
-      _dd.appsec.event_rules.version: 1.3.0,
```

Comment:
it's because we do some test with a modified rule set that we configure though an env var. We need to do this to check the env var doesn't regress, though there maybe some test that could be moved back to the main file, I'll add a jira to check that.